android/app/src/main/kotlin/coop/polypoly/polypod/Preferences.kt

```diff
@@ -7,8 +7,8 @@ import org.json.JSONObject
 class Preferences {
     companion object {
         private const val firstRunKey = "firstRun"
-        private const val seenInAppNotificationIdKey = "seenInAppNotificationId"
-        private const val seenPushNotificationIdKey = "seenPushNotificationId"
+        private const val lastNotificationIdKey = "lastNotificationId"
+        private const val lastNotificationStateKey = "lastNotificationState"
```

Comment:
We don't need to do any migration, since we never showed any notifications in the past - phew :D

Reply:
You're using strings as invalid value for lastNotificationIdKey, I get that, but wouldn't the other value be valid?

Reply:
These are just the **keys**, not the initial values. `SharedPreferences` is essentially a simple key store, I tried to abstract it away a _little_ bit via this `Preferences` class, but it still deals with string keys for each item, hence these constants.